src/main/kotlin/com/reicheltp/celtic_rituals/rituals/bowl/RitualBowlTile.kt

```diff
@@ -21,14 +23,23 @@ import net.minecraftforge.items.ItemStackHandler
  * TileEntity for @see RitualBowlBlock entity.
  */
 class RitualBowlTile : TileEntity(ModBlocks.RITUAL_BOWL_TILE!!), IInventory {
+    public var specialItem: ItemStack = ItemStack.EMPTY
+        private set
+
     // The item handler capability keeps
-    private val ingredients = LazyOptional.of { ItemStackHandler(5) }
+    private val ingredients = LazyOptional.of {
+        object : ItemStackHandler(5) {
+            override fun getSlotLimit(slot: Int): Int = 1
```

Comment:
Should the SlotLimit per Item be equal to 1?

Reply:
Yes, this is intended that you can only add exactly 5 items. No matter if they stack.